services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/models/domains/shared_store.py

```diff
@@ -5,8 +5,9 @@
 
 class SharedStore(BaseModel):
     compose_spec: Optional[str] = Field(
-        None, description="stores the stringified compose spec"
+        default=None, description="stores the stringified compose spec"
```

Comment:
question: what advantage is there when specifying the keyword `default`?

Reply:
pylance complaints if you do not do that. It is something SAN discovered.